sql/synth_recipes.sql

```diff
@@ -2029,6 +2029,13 @@ INSERT INTO `synth_recipes` VALUES (25514,0,0,0,0,118,0,0,0,0,0,4096,4238,3323,8
 -- INSERT INTO `synth_recipes` VALUES (25527,0,2004,0,0,255,0,0,0,0,0,4096,4238,4076,9250,16408,0,0,0,0,0,21513,21514,21514,21514,1,1,1,1,'Raetic Baghnakhs');
 -- INSERT INTO `synth_recipes` VALUES (25528,0,0,0,255,255,0,0,0,0,0,4096,4238,4077,4081,9248,16605,0,0,0,0,21606,21607,21607,21607,1,1,1,1,'Enriching Sword');
 -- INSERT INTO `synth_recipes` VALUES (25529,0,2005,0,0,255,0,255,0,0,0,4102,4244,9003,9062,9251,9832,9844,9845,9845,9852,21628,21629,21630,21630,1,1,1,1,'Valor Sword');
+INSERT INTO `synth_recipes` VALUES (29886,1,0,0,0,71,0,0,0,0,0,4100,4242,12802,0,0,0,0,0,0,0,914,851,931,745,1,1,1,1,'Gold Cuisses (desynth)');
+INSERT INTO `synth_recipes` VALUES (29887,1,0,0,0,77,0,0,0,0,0,4100,4242,12674,0,0,0,0,0,0,0,850,931,931,745,2,2,3,1,'Gold Gauntlets (desynth)');
+INSERT INTO `synth_recipes` VALUES (29888,1,0,0,0,33,0,0,0,0,0,4098,4240,1614,0,0,0,0,0,0,0,760,801,814,815,1,1,1,1,'Corse Bracelet (desynth)');
```

Comment:
You have Chysoberyl for HQ1. should be HQ3

Reply:
This was a dyslexia moment most likely from reading from FFXIAH and Wiki.  Resolved.